PerfectXL.EPPlus/VBA/ExcelVBAProject.cs

```diff
@@ -259,6 +259,10 @@ private void ReadProjectProperties()
                             _protection.VbeProtected = (cmg[0] & 4) != 0;
                             break;
                         case "DPB":
+                            if (!Regex.IsMatch(split[1], "^[\\dA-F]+$"))
+                            {
+                                continue;

```

Comment:
I would say `break` here instead of "continue".

When using `continue` inside the switch, if someone adds code below the `switch` block later, we might have a bug.